src/sickle.ts

```diff
@@ -238,8 +252,8 @@ function last<T>(elems: T[]): T {
   return elems.length ? elems[elems.length - 1] : null;
 }
 
-export function annotate(file: ts.SourceFile): string {
-  return new Annotator().annotate(file);
+export function annotate(file: ts.SourceFile, options?: SickleOptions): string {
+  return new Annotator(options || defaultOptions).annotate(file);
```

Comment:
Ah right.  I should make all the fields on options optional.  I will ping you pack when this is done.